Scheduler/FirstComeFirstServe.c

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file FirstComeFirstServe.c
+ */
+
+#include "FirstComeFirstServe.h"
+#include "ScheduleFile.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+void RunFCFSProcess (ScheduleData* inputData){
+	bool debug = false;		
+	
+	if(debug) printf("Starting FCFS\n");
+
+	FILE *out = fopen("processes.out", "w");
+	if(out == NULL) {
+		printf("Error opening output file\n");
+		return;
+	}
+	if(debug) printf("Opened processes.out\n");
+
+	int numProc = VectorCount(inputData->processes);
+
+	if(debug) printf("%d processes\nUsing First-Come First-Serve\n\n", numProc);
+	fprintf(out, "%d processes\nUsing First-Come First-Serve\n\n", numProc);
+
+	/*
+	* How-to:
+	* Processes are completed in the order they are received.
+	*/
+
+	Vector* queue;
+	VectorCreate(sizeof(InputProcess), numProc, 0, NULL, &queue);
```

Comment:
With the merge with master, the VectorCreate method has been updated to include an extra parameter for compareTo. You need to pass an extra NULL after the first NULL. Your code does not currently compile because of this.